kwok/apis/v1alpha1/kwoknodeclass_status.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/awslabs/operatorpkg/status"
+)
+
+// KwokNodeClassStatus contains the resolved state of the KwokNodeClass
+type KwokNodeClassStatus struct {
```

Comment:
nit: should this be `KWOK` since it's an acronym?

Reply:
That makes sense, but for some reason KwokNodeClass looks better to me from a camelcased perspective. From a titling perspective, you could think it should be acronym'd like KwoK, which is how I've sometimes referenced it, since `wo` is an "unimportant" word. WDYT?

Reply:
I think we generally want to just capitalize acronyms everywhere as a style thing if we can. See: https://go.dev/wiki/CodeReviewComments#initialisms

Reply:
okkkkkkkkkk